sdk/tables/azure-data-tables/README.md

```diff
@@ -25,17 +20,287 @@ best parts of Azure cloud.
 ```
 [//]: # ({x-version-update-end})
 
+### Prerequisites
```

Comment:
IIRC, we have an order of titles for READMEs.. can you confirm? @czubair

Reply:
Yep, I am following the layout documented here: https://review.docs.microsoft.com/en-us/help/contribute-ref/contribute-ref-how-to-document-sdk?branch=master#readme